components/fileqcs/fileQcsController.js

```diff
@@ -204,7 +204,7 @@ const addManyFileQcs = async (req, res, next) => {
 
     const fqcInserts = await addFqcs(hydratedFqcs);
     // merge the results
-    const merged = mergeFileResults(fprs, fqcInserts.fileqcs);
+    const merged = mergeFileResults(fprs, fqcInserts.fileqcs, swids);
```

Comment:
I am a bit confused about why `mergeFileResults` is called with more parameters than it is defined to accept?

Reply:
Oh, that was from an earlier change that didn't make it through to this committed version. Thanks, I'll update it!